docs/Modders/uniques.md

```diff
@@ -18,6 +18,11 @@ Simple unique parameters are explained by mouseover. Complex parameters are expl
 
 	Applicable to: Triggerable, Global
 
+??? example  "Sell [buildingFilter] [cityFilter]"
+	Example: "Sell [Culture] [in all cities]"
```

Comment:
Looks like docs weren't updated with your latest changes:
```suggestion
??? example  "Sell [buildingFilter] buildings [cityFilter]"
	Example: "Sell [Culture] buildings [in all cities]"
```

Reply:
That's fine, version release will update those docs automagically :)